src/main/scala/com/azavea/modellab/Parser.scala

```diff
@@ -45,12 +48,35 @@ class Parser(layerRegistry: LayerRegistry, layerReader: FilteringLayerReader[Lay
   implicit class withJsonMethods(json: JsValue) {
     val fields = json.asJsObject.fields
     def get[T: JsonReader](name: String) = fields(name).convertTo[T]
-    def param[T: JsonReader](name: String) = 
+    def param[T: JsonReader](name: String): T =
```

Comment:
This should throw a helpful error rather than the unhelpful error you've found. Something like "Looking for $param in this $json"